go/vt/mysqlctl/mysqld.go

```diff
@@ -688,7 +687,12 @@ func deleteTopDir(dir string) (removalErr error) {
 // executeMysqlCommands executes some SQL commands,
 // using the mysql command line tool.
 func (mysqld *Mysqld) executeMysqlCommands(connParams *sqldb.ConnParams, sql string) error {
-	return mysqld.executeMysqlScript(connParams, strings.NewReader(sql))
+	params, err := dbconfigs.WithCredentials(connParams)
```

Comment:
The other use case is right after Init, we just want to connect as user root with no password, so we don't want to go through credentials server. I think that makes sense.